boilerplate/ignite/templates/screen/NAME-screen.tsx.ejs

```diff
@@ -8,34 +8,33 @@ import React, { FC } from "react"
 import { observer } from "mobx-react-lite"
```

Comment:
we probably want the screen name to be like `ExampleScreen`, not `example-screen`. Not sure exactly where this needs to change.

Reply:
I think that is a good idea, it would match the contention we already have.

We'd need to add some logic to the [`src/tools/generators.ts`](https://github.com/infinitered/ignite/blob/master/src/tools/generators.ts#L197), which I feel is a little out of scope for this PR. I mainly wanted to get the generator tests passing again in this PR, while I feel this is more of a feature request.

Reply:
Works for me! I was going off of the PR title and assumed haha :)

Reply:
Fair enough, thanks for taking a look!